includes/core-changes/corefx/2.1/instantiate-struct.md

```diff
@@ -43,16 +43,6 @@ if (key.ToString() == "y")
     Console.WriteLine("Yes!");
 ```
 
-```csharp
-ConsoleKeyInfo[] keys = new ConsoleKeyInfo[5];    // Array of structs.
-
-for (int i = 0; i < keys.Length; i++)
-{
-    // Initialize each array element with the new operator.
-    keys[i] = new ConsoleKeyInfo();
-}
-```
```

Comment:
The following code compiles in .NET Core 3.1, so no need to initialize each array element separately:
```csharp
ConsoleKeyInfo[] keys = new ConsoleKeyInfo[5];
for (int i = 0; i < keys.Length; i++)
{
    Console.WriteLine(keys[i]);
}
```

As far as I understand, the elements of a struct array are initialized into the default value of the struct (as the result of allocating zeroed memory for an array and zero-bit patterns being the default value of structs).